apps/server/src/modules/common-cartridge/import/common-cartridge-resource-factory.spec.ts

```diff
@@ -132,5 +147,61 @@ describe('CommonCartridgeResourceFactory', () => {
 				expect(result).toBeUndefined();
 			});
 		});
+
+		describe('when creating a web content resource', () => {
+			const setup = () => {
+				const webContentHtml = setupWebContentHtml();
+				const organizationProps = setupOrganizationProps();
+
+				organizationProps.resourceType = CommonCartridgeResourceTypeV1P1.WEB_CONTENT;
+				admZipMock.getEntry.mockReturnValue({} as AdmZip.IZipEntry);
+				admZipMock.readAsText.mockReturnValue(webContentHtml);
+
+				return { organizationProps };
+			};
+
+			describe('when web content is provided', () => {
+				it('should create a web content resource', () => {
+					const { organizationProps } = setup();
+
+					const result = sut.create(organizationProps);
+
+					expect(result).toStrictEqual({
+						type: CommonCartridgeResourceTypeV1P1.WEB_CONTENT,
+						title: organizationProps.title,
+						html: 'Content',
+					});
+				});
+			});
+
+			describe('when web content is not provided', () => {
+				it('should return an empty value', () => {
+					const { organizationProps } = setup();
+					const emptyWebContent = '';
+					admZipMock.readAsText.mockReturnValue(emptyWebContent);
+
+					const result = sut.create(organizationProps);
+
+					expect(result).toStrictEqual({
+						type: CommonCartridgeResourceTypeV1P1.WEB_CONTENT,
+						title: organizationProps.title,
+						html: '',
+					});
+				});
+			});
+
+			describe('when html is not valid', () => {
+				it('should return undefined', () => {
+					const { organizationProps } = setup();
+
+					// eslint-disable-next-line @typescript-eslint/no-explicit-any

```

Comment:
Avoid casting and usage of disabling the linter. This can be a possible solution:

```typescript
const tryCreateDocumentMock = jest.fn().mockReturnValue(undefined);
Reflect.set(sut, 'tryCreateDocument', tryCreateDocumentMock);
```

Reply:
done ✅